File: padroesCriacionais/padrao-de-projeto-abstractFactory/php/tests/unit/EasyMapTest.php
<?php

namespace Abstractfactory\tests\unit;

use Abstractfactory\Php\EasyMap;
use PHPUnit\Framework\TestCase;

class EasyMapTest extends TestCase
{
    public function testShouldReturnEasyMap(): void
    {
        $easyMap = new EasyMap();
        $this->assertEquals("gera mapa no modo facil", $easyMap->makeMap());
    }
}

File: padroesCriacionais/padrao-de-projeto-abstractFactory/java/src/test/java/EasyMapTest.java
import abstractFactory.EasyMap;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class EasyMapTest {

    @Test
    public void shoudlReturnEasyMap() {
        EasyMap easyMap = new EasyMap();
        assertEquals("gera mapa no modo facil", easyMap.makeMap());
    }
}
